mssql-mixin/config.libsonnet

```diff
@@ -1,5 +1,7 @@
 {
   _config+:: {
+    enableMultiCluster: false,
+    mssqlSelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
```

Comment:
Missing `multiClusterSelector: 'job=~"$job"'` in the config, this might still be missing from the docs. I made a PR a while back but I'm not sure it got merged.